index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="Your description goes here">
+    	<meta name="keywords" content="one, two, three">
+
+		<title>Relaxr</title>
+
+		<!-- external CSS link -->
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,800,300,300italic' rel='stylesheet' type='text/css'>
+
+	</head>
+	<body>
+		<div class="container">
+
+			<div class='intro'>
+	
+					<p class='logo' id='logotop'>
+						Relaxr
+					</p>
+
+					<h2>
+						Get piece of mind with a single tap
+					</h2>
+
+					<button class=button id="getit">
+						Get it Now
+					</button>
+
+			</div>
+
+			<div id="info">
```

Comment:
please fix your indentation settings to double spaces as we went over in class
if you are not sure please come and see me